cypress/e2e/signIn.cy.js

```diff
@@ -10,29 +10,31 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  const wrongEmail = '123' + user.email;
```

Comment:
The user object is undefined when 'wrongEmail' is declared. Consider moving the 'wrongEmail' declaration inside the 'beforeEach' hook after the user object is defined.